### ISSUE TITLE
README.md

### ISSUE THREAD

**Issue by WiseK** [WillyMoselhy/AVDReplacementPlans#23]
Consolidate instructions or move additional information like RBAC to Readme

**Comment by WillyMoselhy** [comment]
I agree, I will move all setup instructions under a different path. Maybe a Wiki?
